app/experimenter/experiments/serializers.py

```diff
@@ -35,6 +35,22 @@ def to_representation(self, obj):
             return obj
 
 
+class VariantsListSerializer(serializers.ListSerializer):
+
+    def to_representation(self, data):
+        data = super().to_representation(data)
+        if data == []:
+            blank_variant = {}
+            for field in self.child.fields:
+                blank_variant[field] = None
+                control_blank_variant = blank_variant.copy()
+                blank_variant["is_control"] = False
```

Comment:
If this is inside the loop doesn't it do the same thing on every iteration?

Reply:
oooh some bad indenting.... 🙁